packages/schema/bind/src/bindings/wasm-as/index.ts

```diff
@@ -44,13 +44,13 @@ export function generateBinding(typeInfo: TypeInfo): OutputDirectory {
 
   // Generate imported folder
   if (
-    typeInfo.importedQueryTypes.length > 0 ||
+    typeInfo.importedmoduleTypes.length > 0 ||
```

Comment:
This is not correct “camel casing”, should be “importedModuleTypes” not “importedmoduleTypes”